docs/decisions/0009_pii.rst

```diff
@@ -0,0 +1,71 @@
+9. Personally-identifying information (PII)
+###########################################
+
+Status
+******
+
+Accepted
+
+Context
+*******
+
+The vast majority of event data processed by Aspects is anonymized to protect user privacy. Most events have an "actor",
+and that actor is uniquely identified by an anonymous user ID.
+
+But some of the community's analytics use cases call for the use of Personally Identifying Information (PII) that is
+stored in Open edX. For example, to identify and intercede with learners that are struggling in a course, we need to see
+the contact details for that learner. Or if we want to use use demographic data in recruitment campaigns to improve the
+diversity of a student group, we need access to user profile fields like country, state, gender, and age group.
+
+Storing and displaying PII must be done with care, so this document describes the steps needed to protect this data and
+help institutions to use it responsibly.
+
+Decision
+********
+
+**Opt-in Aspects PII**
```

Comment:
Sadly the v1 reqs will have this on by default to support the learner charts, so this will become opt-out in the near future.

Reply:
Ah ok.. [5e8092d](https://github.com/openedx/openedx-aspects/pull/106/commits/5e8092dd7b52826dc423a3d3fcad1860692fa8aa) flips that bit.